iceoryx_hoofs/test/moduletests/test_unit_duration.cpp

```diff
@@ -1505,10 +1533,10 @@ TEST(Duration_test, AddDurationDoesNotChangeOriginalObject)
     constexpr Duration EXPECTED_DURATION{13_s + 42_ns};
 
     auto sut1 = EXPECTED_DURATION;
```

Comment:
Why isn't this const?

Reply:
clang-tidy did not complain and changing this all over the place would take quite an amount of time with little gain. After all this is test code.